packages/runtime/container-runtime/src/messageTypes.ts

```diff
@@ -237,3 +256,19 @@ export type InboundSequencedContainerRuntimeMessageOrSystemMessage =
  */
 export type InboundSequencedRecentlyAddedContainerRuntimeMessage = ISequencedDocumentMessage &
 	Partial<RecentlyAddedContainerRuntimeMessageDetails>;
+
+/**
+ * @alpha
+ * @legacy
+ */
+export interface IExternalContainerRuntimeMessage {
+	type: string;
+
+	contents: unknown;
+}
+
+/**
+ * @alpha
+ * @legacy
+ */
+export type TExternalContainerRuntimeMessage = IExternalContainerRuntimeMessage;
```

Comment:
What is the reason we need this type? I.e. why IExternalContainerRuntimeMessage is not sufficient?
Can you please add some description over for these types?